kratos/geometries/nurbs_curve_tessellation.h

```diff
@@ -0,0 +1,161 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Thomas Oberbichler
+//
+//  Ported from the ANurbs library (https://github.com/oberbichler/ANurbs)
+//
+
+#if !defined(KRATOS_NURBS_CURVE_TESSELLATION_H_INCLUDED )
+#define  KRATOS_NURBS_CURVE_TESSELLATION_H_INCLUDED
+
+#include "nurbs_curve_geometry.h"
+
+#include "includes/ublas_interface.h"
+#include "containers/array_1d.h"
+
+namespace Kratos {
+
+template <typename TCurve>
+class NurbsCurveTessellation
+{
+public:     // types
+    using Vector = array_1d<double, TCurve::GetDimension()>;
```

Comment:
Kratos has already it's own type of Vector. I think this would mix things up here...

Reply:
I just need some kind of n-dimensional vector here

Reply:
You can call it Array1dType. But vector is definiteley misleading.